### PR TITLE
Linting pages

### PR DESCRIPTION
**Pull request by g1st**
Find it tricky to solve ```is already declared in the upper scope  no-shadow``` kind of errors.

### CODE REVIEW

File: pages/about.js [hunk 1]
```diff
@@ -48,12 +48,14 @@ const styles = theme => ({
 
 class About extends React.Component {
   _renderItem = item => {
-    const onImageError = this.props.onImageError || this._handleImageError;
+    const { onImageError: imageError, onImageLoad, width } = this.props;
+
+    const onImageError = imageError || this._handleImageError;
```
Comment (g1st) [inline pages/about.js:54]: Actually I think onImageError and onImageLoad are never passed from props and not used here at all. Did not want to make any changes apart linting.

Comment (nikrb) [reply pages/about.js:54]: agreed. easier to take it out later than work out where they come from

**Review by nikrb — APPROVED** [review]
LGTM

File: pages/about.js (hunk 1 above)
Comment (zurda) [reply pages/about.js:54]: I created an issue for this, to investigate later. Resolving this for now.

**Pull request merged**